### PR TITLE
Media whitelist checks proper extension and is case insensitive

### PR DESCRIPTION
**Pull request by stowellm**
Addresses #2061

### CODE REVIEW

**Comment by kcarcia** [comment]
@stowellm This passes my code review (nice work simplifying that line :) ), but my imaginemagick doesn't work, so I'm probably not the best person to review this. I'm going to re-assign it to @jaypoulz

File: app/controllers/media_objects_controller.rb [hunk 1]
```diff
@@ -70,8 +70,9 @@ def destroy
 
   # POST /media_object/saveMedia
   def saveMedia
-    extension = params[:upload].original_filename.split('.')[1]
-    unless FileUploader.upload_whitelist.include? extension
+    extensions = params[:upload].original_filename.split('.')
```
Comment (jaypoulz) [inline app/controllers/media_objects_controller.rb]: ``extension = params[:upload].original_filename.split('.')[-1]``
Is what you're looking for

**Comment by jaypoulz** [comment]
Just the one thing, then mark as reviewed

**Comment by stowellm** [comment]
Fixed the thing, marked as reviewed as Jeremy requested.  Assigning to Kate to merge when appropriate.

**Pull request merged**